redirects.yaml

```diff
@@ -745,7 +745,6 @@ telcos/?: "/telecommunications"
 
 # Blog redirects
 blog/search/?: /search
-blog(?P<page>(/.+)?)/feed?: "https://admin.insights.ubuntu.com{page}/feed"
```

Comment:
Could you also change lines `607`, `608` and `685` of `redirects.yaml`? So that we no longer send anyone to `admin.insights.ubuntu.com`?